extensions/atomic/README.md

```diff
@@ -0,0 +1,26 @@
+# Atomic-Raycast
+
+Plugin for Raycast to search your [Atomic Server](https://github.com/joepio/atomic-data-rust/).
+
+Features
+
+- [x] Use the `/search` endpoint for an Atomic Server
+- [x] Titles, descriptions and class names are previewed
+- [x] Markdown of descriptions is rendered
+- [x] Open in browser action
+- [x] Set your server on startup, save as preference
+- [ ] Other cool features? Open an issue or PR!
+
+## Installing
+
+_Note: this extension is beta and not yet available in the Raycast store_
```

Comment:
Do you want to remove this line before I merge and publish the extension?

Reply:
Yes I do, thanks! I've updated the instructions.